bears/kotlin/KotlinLintBear.py

```diff
@@ -0,0 +1,26 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='ktlint',
+        global_bear=True,
+        output_format='regex',
+        output_regex=r'(?P<filename>.*):(?P<line>\d+):'
```

Comment:
dont capture `filename` as it is ignored anyway -- it is always equal to the filename given to the linter.